dali/ft/fttransform.hpp

```diff
@@ -31,6 +31,7 @@ interface ITransformer : public IInterface
     virtual void setInputCRC(crc32_t value) = 0;
     virtual bool setPartition(RemoteFilename & remoteInputName, offset_t _startOffset, offset_t _length, bool compressedInput, const char *decryptKey) = 0;
     virtual offset_t tell() = 0;
+    virtual unsigned __int64 getStatistic(StatisticKind kind) = 0;
```

Comment:
Is this function ever called?

Reply:
yes, this is called in TransferServer::appendTransformed by "input->getStatistic(StNumDiskReads)".  (line 545 fttransform.cpp)